models/user_model.go

```diff
@@ -9,5 +12,10 @@ type User struct {
 	EmailAddress     string `bson:"email_address" json:"email_address" binding:"required"`
```

Comment:
Can we decide on using camel case for json? Seems like snake case is a bit out of place and [google recommends camel case](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format).